src/java.base/share/classes/java/lang/foreign/SymbolLookup.java

```diff
@@ -129,6 +129,30 @@ public interface SymbolLookup {
      */
     Optional<MemorySegment> find(String name);
 
+    /**
+     * {@return a composite symbol lookup which will retrieve symbols using both this and the provided symbol
+     * lookup} More specifically, if a symbol is not found using this lookup, the provided lookup will be
+     * used as fallback. In other words, the resulting symbol lookup will only return {@linkplain Optional#empty()}
+     * if both lookups fail to retrieve a given symbol.
```

Comment:
```suggestion
     * {@return a composed symbol lookup that returns result of finding the symbol with this lookup if found, otherwise returns the result of finding the symbol with the other lookup.}
```

Reply:
Not very IHMO. I was just trying to say the same thing you said with less words, which makes it more amenable to being expressed as a compact javadoc header.

Reply:
ok - thanks!